docs/installation/teams/index.md

```diff
@@ -224,10 +224,40 @@ If you get 404, please check the ingress configuration or endpoint you configure
    If Botkube responds, send `@Botkube notifier start` to enable notifications.
    ![Teams - check bot health](assets/teams_ping.png "Teams Botkube ping")
 
+   :::warning
```

Comment:
this is an alias for `danger`. WDYT about:

```suggestion
   :::caution
```

It will look like this:
![image](https://user-images.githubusercontent.com/7155799/203923796-b51cb156-5946-49ef-a3bd-8b4483db6b2f.png)